model/src/main/kotlin/Project.kt

```diff
@@ -132,4 +132,24 @@ data class Project(
             vcsPath = vcsPath
     )
 
+    companion object {
+        /**
+         * Creates an instance of [Project] where all properties are empty strings or empty collections.
+         */
+        fun createEmptyProject() =
```

Comment:
Maybe omit the `Project` suffix since this already is in the `Project` class? Or create a top-level function `emptyProject()` similar to Kotlin's `emptyList()` etc.?